solutions/21.js

```diff
@@ -0,0 +1,7 @@
+const solution = (node, nextNode = node.next, max = node.next) => {
+  if (nextNode == null) return max;
```

Comment:
In the `if` statement condition, I would do `nextNode == null || nextNode == node` in case if there is a loop linked list.

Reply:
Great detail to add. I just made those changes. Thanks E-man.